BookingsApi/BookingsApi.DAL/Migrations/20210216172749_AddGeneralHearingTypes.cs

```diff
@@ -0,0 +1,92 @@
+﻿using BookingsApi.Domain.Enumerations;
+using BookingsApi.Domain.RefData;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace BookingsApi.DAL.Migrations
+{
+    public partial class AddGeneralHearingTypes : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.InsertData(
+                table: nameof(HearingType),
+                columns: new[] { "Id", "Name", "CaseTypeId" },
+                values: new object[,]
+                {
+                    { 141, "Daily Test", 3 },
+                    { 142, "Demo", 3 },
+                    { 143, "Familiarisation", 3 },
+                    { 144, "One to one", 3 },
+                    { 145, "Test", 3 },
```

Comment:
please can you add 'Automated Test'? Emma has updated the story

Reply:
Done